src/trans-netlist/trans_trace.cpp

```diff
@@ -165,6 +165,30 @@ void show_trans_state(
 
 /*******************************************************************\
 
+Function: show_trans_trace
+
+  Inputs:
+
+ Outputs:
+
+ Purpose:
+
+\*******************************************************************/
+
+void show_trans_trace(
+  const trans_tracet &trace,
+  messaget &message,
+  const namespacet &ns,
+  std::ostream &out)
+{
+  auto l = trace.get_min_failing_timeframe().value_or(trace.states.size() - 1);
```

Comment:
Is there a possibility of `trace.states.empty()`?

Reply:
Shouldn't, but I'll add a `PRECONDITION`